apps/projects/routes.py

```diff
@@ -913,6 +924,38 @@ class RetrieveOrCreateThumbnails(MethodView):
             'type': 'float',
             'coerce': float,
         },
+        'crop': {
+            'type': 'dict',
+            'coerce': literal_eval,  # crop args are a string represent of a dict
+            'required': False,
+            'empty': True,
+            'schema': {
+                'width': {
+                    'type': 'integer',
```

Comment:
IMO we should add `min` here and for `height`. For example I can send a request to generate a preview thumbnail 5x5 and it will generate it, but I can also send a request to generate a thumbnail 1x1 and in this case ffmpeg dies :)

Reply:
It seems ffmpeg will fail only with either width or height equals 1 so I think min is 2 is safe

Reply:
I think we can put min to `MIN_VIDEO_WIDTH` and `MIN_VIDEO_HEIGHT`